fftoml/fftoml.go

```diff
@@ -6,27 +6,69 @@ import (
 	"io"
 	"strconv"
 
-	"github.com/BurntSushi/toml"
+	"github.com/pelletier/go-toml"
 	"github.com/peterbourgon/ff"
 )
 
 // Parser is a parser for TOML file format. Flags and their values are read
 // from the key/value pairs defined in the config file.
 func Parser(r io.Reader, set func(name, value string) error) error {
-	var m map[string]interface{}
-	_, err := toml.DecodeReader(r, &m)
-	if err != nil {
-		return ParseError{Inner: err}
+	return ParserWith()(r, set)
+}
+
+type config struct {
```

Comment:
I'd rather see this type as exported, so Config, to avoid having functions in the package's API that operate on unexported types